primitives/weights/src/lib.rs

```diff
@@ -121,6 +121,46 @@ pub struct WeightToFeeCoefficient<Balance> {
 /// A list of coefficients that represent one polynomial.
 pub type WeightToFeeCoefficients<T> = SmallVec<[WeightToFeeCoefficient<T>; 4]>;
 
+/// A list of coefficients that represent a polynomial.
```

Comment:
What is the formula for the polynomial? It's hard to know what the coefficients stand for without it...